config/config.go

```diff
@@ -0,0 +1,50 @@
+// File: config/config.go
```

Comment:
- The `LoadConfig` function's comment should be separated by a blank line from the comment above it, which describes the file.
- Ensure the second sentence in the `LoadConfig` function's comment starts with a capital letter for consistency: "and unmarshals them into a Config struct" should be "And unmarshals them into a Config struct."

Overall, the file is fairly clean, but just needs these minor adjustments to align perfectly with the style guide.